pkg/descheduler/strategies/pod_antiaffinity_test.go

```diff
@@ -149,12 +156,26 @@ func TestPodAntiAffinity(t *testing.T) {
 			nodes:                   []*v1.Node{node1, node3},
 			expectedEvictedPodCount: 0,
 			nodeFit:                 true,
+			testFunc:                func(podList []v1.Pod) {},
+		},
+		{
+			description:             "No pod to evicted since all pod terminating",
+			maxPodsToEvictPerNode:   0,
+			pods:                    []v1.Pod{*p1, *p2, *p3, *p4},
+			nodes:                   []*v1.Node{node1},
+			expectedEvictedPodCount: 3,
+			testFunc: func(podList []v1.Pod) {
+				for _, pod := range podList {
+					pod.DeletionTimestamp = &metav1.Time{}
```

Comment:
This actually modifies the pods permanently. Each test needs a freshly created pod. Instead, can you create new pods (e.g. p5 - p8) which have the `DeletionTimestamp` set right away?

Reply:
Thanks~ I learn it. I can not check that testcase need a freashly created pod. So I would create a new pod for this test case, is it ok for you?

Reply:
Fix done~ Please check :)

Reply:
@JaneLiuL if you are creating p9 and p10 now specifically for this test, do you still need `testFunc` at all? you can just specify those pods with an empty timestamp

Reply:
Already remove `testFunc`, may I click `Resolve conversation`?